dockstore-webservice/src/main/java/io/openapi/api/impl/ServiceInfoApiServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package io.openapi.api.impl;
+
+import java.util.Date;
+import java.util.Optional;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+import io.dockstore.webservice.core.User;
+import io.openapi.api.ServiceInfoApiService;
+import io.openapi.model.Service;
+import io.openapi.model.ServiceType;
+
+public class ServiceInfoApiServiceImpl extends ServiceInfoApiService {
+    @Override
+    public Response getServiceInfo(SecurityContext securityContext, ContainerRequestContext value, Optional<User> user) {
+        Service service = getService();
+        return Response.ok().entity(service).build();
+    }
+
+    public static Service getService() {
+        Service service = new Service();
+        service.setId("1");
+        service.setName("Dockstore");
+        service.setType(getServiceType());
+        service.setDescription("The GA4GH Tool Registry API is a standard for listing and describing available tools (both stand-alone, Docker-based tools as well as workflows in CWL, WDL, Nextflow, or Galaxy) in a given registry,");
+        service.setOrganization(getOrganization());
+        service.setContactUrl("https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506");
+        service.setDocumentationUrl("https://docs.dockstore.org/en/develop");
+        // TODO: Somehow get swagger codegen to generate string parameter (RFC 3339) instead of Date
+        service.setCreatedAt(new Date());
+        // TODO: Somehow get swagger codegen to generate string parameter (RFC 3339) instead of Date
+        service.setUpdatedAt(new Date());
+        service.setEnvironment("prod");
+        service.setVersion("2.0.1");
```

Comment:
Should at least be a constant (you have it below). Or, without fully understanding the whole TRS build process as it's integrated with Dockstore, could we derive the version from the openapi.yaml that we build against? I don't know if the version is in there or not. But when we upgrade to 2.0.2, we'll have to remember to update this version.

Reply:
Ok, I think the version above is supposed to be Dockstore version (1.10.0-SNAPSHOT)

Reply:
Use Maven filtering to auto populate this for auto updates. https://github.com/dockstore/dockstore/blob/develop/dockstore-webservice/src/main/templates/io/dockstore/webservice/resources/Description.java has an example